__tests__/app.test.js

```diff
@@ -37,8 +41,18 @@ describe.only("GET /api/topics", () => {
         })
         
     });
-}) 
+})
 
+describe("GET /api endpoints ", () => {
+    test('Should respond with an object of available endPoints and their useful info', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+            expect(response.body).toEqual({ endPoints })
```

Comment:
Good use of the required in variable for this test - this will keep it dynamic to whatever endpoints.json looks like